tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -210,10 +210,7 @@ def check_model_object(argv):
         return 'onnx'
 
     if 'paddle' in sys.modules:
-        import paddle
-        if isinstance(model, paddle.hapi.model.Model) or isinstance(model,
-                                                                    paddle.fluid.dygraph.layers.Layer) or isinstance(
-            model, paddle.fluid.executor.Executor):
+        if paddle_model_check_instance(model, "paddle.hapi.model.Model") or paddle_model_check_instance(model, "paddle.nn.layer.layers.Layer") or paddle_model_check_instance(model, "paddle.fluid.dygraph.layers.Layer") or paddle_model_check_instance(model, "paddle.base.Executor") or paddle_model_check_instance(model, "paddle.fluid.executor.Executor"):
```

Comment:
Can we make a list of class names ["paddle.hapi.model.Model", ...] and iterate over them?

Reply:
It's done.